app/src/views/campaigns/admin/internships.vue

```diff
@@ -125,6 +131,34 @@
       </el-table-column>
     </el-table>
 
+    <el-dialog
+      :title="$t('table.mentoringProposition.title')"
```

Comment:
It might be good to also change title when we are under update mode to think like "Modification de la proposition"